src/logging_unittest.cc

```diff
@@ -1075,13 +1075,14 @@ TEST(Strerror, logging) {
 
 // Simple routines to look at the sizes of generated code for LOG(FATAL) and
 // CHECK(..) via objdump
+/*
 static void MyFatal() {
   LOG(FATAL) << "Failed";
 }
 static void MyCheck(bool a, bool b) {
   CHECK_EQ(a, b);
 }
-
+*/
```

Comment:
What's the reason for keeping the functions in a comment?

Reply:
I was thinking if someone might want to use them in future. If you think they should be removed, please let me know